src/api/app/views/webui2/webui/search/_breadcrumb_items.html.haml

```diff
@@ -0,0 +1,8 @@
+= render partial: 'webui/main/breadcrumb_items'
+
+%li.breadcrumb-item.active{ 'aria-current' => 'page' }
+  - case action_name
```

Comment:
This could be simplified with a `if`, because the only different part is `Owners`. However, this is just too much in my opinion. So I keep it as it is.